docs/modules/haproxy.md

```diff
@@ -1,8 +1,10 @@
 # HAProxy
 
 ### Purpose
+[HAProxy](http://www.haproxy.org/) is a high performance TCP/HTTP load balancer and proxying solution. The common use for HAProxy is to improve performance and reliability of a server environment by distributing the workload across several servers; web, data, etc.
 
 ### Dependencies
+HAProxy can be installed from public repositories through APT and YUM. `sudo apt-get install haproxy` for Ubuntu or `sudo yum install haproxy` for CentOS/RHEL. HAProxy can also be built from source. Downloads are available through their [website](http://www.haproxy.org/#down). More information can be found in their [documentation](http://www.haproxy.org/#doc1.7)
```

Comment:
I feel like it would make more sense to generalize and not call out 'apt' and 'yum' by name. What if I'm using 'apk', 'dnf', 'pacman', 'dpkg', 'rpm', 'slackpkg', 'portage', 'emerge', or 'abs'? It might be more apt (HAHA) to state merely that a 'package manager' can be used. The same goes for installation instructions, it might not make sense to zero in on specific install steps for a small subset of the available distribution, rather we should attempt to provide a broad look at what is necessary to get a module running on any Linux system.